### ISSUE TITLE
coreos-growpart: sometime multiple sysroot mounts are detected

### ISSUE THREAD

**Issue by lucab** [coreos/fedora-coreos-config#240]
We just observed a kola run on FCOS which failed in the initramfs, with the following logged on the emergency console:
```
[    5.585822] coreos-growpart[638]: cat: '/sys/dev/block/252:4'$'\n''252:4/partition': No such file or directory
```

**Comment by lucab** [comment]
@jlebon suspects that `findmnt -nvr -o MAJ:MIN /sysroot` somehow got the following:

```
252:4
252:4
```

A bind-mount could indeed result in that output from `findmnt`.

**Comment by jlebon** [comment]
Fixed by #241. (That is, the script handles this case now, though it's still not entirely clear to me where the second `/sysroot` mount comes from).
